Centaurus.Test.Utils/TestEnvironment.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Centaurus.Test
+{
+    public static class TestEnvironment
+    {
+        public const string AlphaSecret = "SD7DAXWXRCVQKGPCATD3JSCWYUEDV2KTH6PLKPSRJZABH4TW4ZDLMNI5";
```

Comment:
Always generate new keypairs in a static constructor to avoid "hardcoded constants" problem.

Reply:
We can't generate it. Static properties can't be used as test case parameters.

Reply:
We can. Check https://github.com/nunit/docs/wiki/TestCaseSource-Attribute#form-3---testcasesourcetype-sourcetype